roles/etcd/templates/etcd-docker.service.j2

```diff
@@ -4,7 +4,7 @@ Wants=docker.socket
 After=docker.service
 
 [Service]
-User=root
+User=etcd
 PermissionsStartOnly=true
 EnvironmentFile=-/etc/etcd.env
 ExecStart={{ bin_dir }}/etcd
```

Comment:
etcd here is a script that call `docker`, so you just broke `etcd_deployment_type: docker`
https://github.com/kubernetes-sigs/kubespray/blob/master/roles/etcd/templates/etcd.j2

Reply:
fix only for: when: etcd_deployment_type == "host"